src/StaticCaching/Cachers/AbstractCacher.php

```diff
@@ -216,10 +217,12 @@ protected function invalidateWildcardUrl($wildcard)
         // Remove the asterisk
         $wildcard = substr($wildcard, 0, -1);
 
+        $wildcard = URL::makeRelative($wildcard);
```

Comment:
Unless I'm misunderstanding, this would make the URLs you specify in the config file all relative, so you'd lose the multi-domain aspect of what I explained in the last comment.

> Then we can have support for invalidating multi-domain urls by telling people to define absolute urls in their config.

Reply:
You're right, this did not work.